buildSrc/src/main/groovy/io.micronaut.build.internal.problem-json-base.gradle

```diff
@@ -0,0 +1,13 @@
+repositories {
+    repositories {
+        mavenCentral()
+        maven { setUrl("https://s01.oss.sonatype.org/content/repositories/snapshots/") }
+    }
+}
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        substitute(module("org.codehaus.groovy:groovy"))
+                .using(module("org.apache.groovy:groovy:${libs.versions.groovy}"))
+    }
+}
```

Comment:
We shouldn't need this any more 🤔

Reply:
Pushed a fix https://github.com/micronaut-projects/micronaut-problem-json/pull/212/commits/78166cb5acb029afb2b817e696004448eb6ac93d